.travis.yml

```diff
@@ -12,8 +12,10 @@ before_script:
   - docker pull selenium/standalone-chrome:2.48.2
   - docker images --no-trunc
 install:
+  - npm install
+  - npm ls || true
```

Comment:
It seems that `npm install` was no longer run, as `install` was overridden in #1140, but tests passed due to cached packages.

Reply:
Interesting.  I guess `check-dependencies` would've caught any difference and just failed the build.  Thanks for fixing this.